bundles/org.eclipse.mita.cli/src/org/eclipse/mita/cli/loader/StandaloneLibraryProvider.xtend

```diff
@@ -0,0 +1,41 @@
+package org.eclipse.mita.cli.loader
+
+import org.eclipse.emf.ecore.resource.Resource
+import org.eclipse.emf.ecore.resource.ResourceSet
+import org.eclipse.mita.cli.commands.CompileToCAdapter
+import org.eclipse.mita.base.scoping.ILibraryProvider
+
+class StandaloneLibraryProvider implements ILibraryProvider {
+	protected ResourceSet resourceSet;
+
+	static private ILibraryProvider instance = new StandaloneLibraryProvider();
+	
```

Comment:
Maybe add a private `new()` and no public ones since this is really a singleton instance